README.md

```diff
@@ -33,4 +33,4 @@ Thank you to all the people who already contributed to ALX SE Theme Selector!
 ALX SE Theme Selector is licensed under the [MIT](https://opensource.org/licenses/MIT) License. See the [LICENSE documentation](https://github.com/vinsky001/ALX-DarkMode-project/blob/main/LICENSE.md) for more information.
 
 ## Acknowledgements
-ALX SE Theme Selector was built by a selection of ALX SE Students. Special thanks to [Elvis](https://github.com/vinsky001) and [Julien](https://github.com/jbarbier).
\ No newline at end of file
+ALX SE Theme Selector was built by a selection of ALX SE Students. Special thanks to [Elvis](https://github.com/vinsky001) and [Julien](https://github.com/jbarbier).
```

Comment:
I believe the second word, `SE`, has to be changed to `Intranet` as well

From
`ALX SE Theme Selector was built by a selection of ALX SE Students`
To
`ALX Intranet Theme Selector was built by a selection of ALX SE Students`